js/layout-javascript/simple-list-code.js

```diff
@@ -2163,10 +2168,25 @@ DynamicList.prototype.updateCommentCounter = function(options) {
   _this.$container.find('.simple-list-comemnt-holder-' + id).html(html);
 }
 
-DynamicList.prototype.showComments = function(id) {
+DynamicList.prototype.showComments = function(id, commentId) {
   var _this = this;
 
   _this.$container.find('simple-list-comment-area').html(_this.commentsLoadingHTML);
+  $('body').addClass('lock');
+  _this.$container.find('.simple-list-detail-overlay-content-holder').addClass('lock');
+  _this.$container.find('.simple-list-comment-panel').addClass('open');
+
+  var context = {
+    dynamicListOpenId: id
+  };
+
+  if (commentId) {
+    context.dynamicListCommentId = commentId;
+  } else {
+    context.dynamicListOpenComments = 'true';
```

Comment:
what's the reason behind using a string here? make a comment if necessary

Reply:
See https://github.com/Fliplet/fliplet-widget-dynamic-lists/pull/353/commits/a345c956b85ce971a4385434350b07503ff72823 This is from what we've been doing for a couple of years with the LFD, so instead of highlighting the reason for this one line, I've added to the top of the query parser file.